website_sale_product_minimal_price/tests/test_website_sale_product_minimal_price.py

```diff
@@ -6,13 +6,14 @@
 class WebsiteSaleProductMinimalPriceHttpCase(HttpCase):
     def setUp(self):
         super().setUp()
-
+        # Always work with dollars
+        self.company = self.env.ref("base.main_company")
+        self.company.currency_id = self.env.ref("base.USD")
```

Comment:
Isn't this going to clash with the currency change constrains?

Reply:
In fact this is forbidden in code under certain conditions, so we will have another problem in the future.

Reply:
Ok, but then what is the best solution? AFAICS the problem is that the test creates a product and assigns a price (not specifying currency) but, when running the tour, the price displayed in the shop follows the default currency set. 
With only this module installed, everything is computed in dollars (default) but, when another module is installed that changes the default currency for the company, it ends up setting a price in euros, but displaying it in dollars (with different values) thus making the tour fail...

Reply:
Create a pricelist with that currency and assign it.